static/src/stylesheets/inline/article-explore.scss

```diff
@@ -134,153 +118,204 @@ $explore-series-header-element-opacity: .85;
         }
     }
 
-    @supports (object-fit: cover) and (height: 1vh) and (display: flex) {
-        top: 319px;
-
-        @include mq($from: tablet) {
-            top: 314px;
-        }
-    }
-
     &.playing {
         opacity: 0;
     }
 }
 
 .explore-series-headline__content {
-    @include fs-headline(5);
-    position: absolute;
-    font-weight: 200;
+    @include fs-headline(4);
+    font-weight: 900;
     color: #ffffff;
     margin: $gs-baseline 0;
-    padding-left: $gs-gutter;
-    width: gs-span(4);
+    width: 94%;
+    padding-left: 3%;
 
     @include mq($from: tablet) {
-        @include fs-headline(8);
-        font-weight: 200;
-        width: gs-span(7);
-        margin: $gs-baseline $gs-gutter*2;
+        @include fs-headline(6);
+        font-weight: 900;
+        width: gs-span(8);
+        padding-left: $gs-gutter;
     }
 }
 
-.explore-series-headline__background {
-    position: absolute;
-    width: 100%;
-}
 
 .inline-explore-series-header-bg__svg {
     fill: $news-main-2;
     opacity: $explore-series-header-element-opacity;
     width: 100%;
 }
 
-.series-identity {
-    position: relative;
-    max-width: gs-span(3.5);
-    top: $gs-baseline*3;
+.explore-series-trail-wrapper {
+    position: absolute;
+    width: 100%;
+    bottom: 0px;
+}
+
+.explore-series-standfirst {
+    background-color: $news-main-2;
+    -webkit-clip-path: polygon(0 0, 100% 15%, 100% 100%, 0% 100%);
+    clip-path: polygon(0 0, 100% 15%, 100% 100%, 0% 100%);
 
     @include mq($from: tablet) {
-        top: 0;
+        -webkit-clip-path: polygon(0 0, 100% 25%, 100% 100%, 0% 100%);
```

Comment:
Should this get picked up by `autoprefixer` ? can you try running `make compile` rather than `make watch` locally, cos `compile` will always run `autoprefixer`. Same for [L165](https://github.com/guardian/frontend/pull/14515/files#diff-1f838bb39bceaff04529b92751ccd71aR165)